tests/page-objects/default/enketo/delivery.wdio.page.js

```diff
@@ -72,147 +72,227 @@ const babyBlueSkinField = (value) =>
 };
 
 const selectDeliveryBabyInfectedUmbilicalCord = async (value) => {
-  return (await babyInfectedUmbilicalCordField(value)).click();
+  const infectedUmbilicalCord = await babyInfectedUmbilicalCordField(value);
+  await infectedUmbilicalCord.waitForClickable();
+  await infectedUmbilicalCord.click();
 };
 
 const selectDeliveryBabyConvulsion = async (value) => {
-  return (await babyConvulsionField(value)).click();
+  const convulsion = await babyConvulsionField(value);
+  await convulsion.waitForClickable();
+  await convulsion.click();
 };
 
 const selectDeliveryBabyDifficultyFeeding = async (value) => {
-  return (await babyDifficultyFeedingField(value)).click();
+  const difficultyFeeding = await babyDifficultyFeedingField(value);
+  await difficultyFeeding.waitForClickable();
+  await difficultyFeeding.click();
 };
 
 const selectDeliveryBabyVomit = async (value) => {
-  return (await babyVomitField(value)).click();
+  const vomit = await babyVomitField(value);
+  await vomit.waitForClickable();
+  await vomit.click();
 };
 
 const selectDeliveryBabyDrowsy = async (value) => {
-  return (await babyDrowsyField(value)).click();
+  const drowsy = await babyDrowsyField(value);
+  await drowsy.waitForClickable();
+  await drowsy.click();
 };
 
 const selectDeliveryBabyStiff = async (value) => {
-  return (await babyStiffField(value)).click();
+  const stiff = await babyStiffField(value);
+  await stiff.waitForClickable();
+  await stiff.click();
 };
 
 const selectDeliveryBabyYellowSkin = async (value) => {
-  return (await babyYellowSkinField(value)).click();
+  const yellowSkin = await babyYellowSkinField(value);
+  await yellowSkin.waitForClickable();
+  await yellowSkin.click();
 };
 
 const selectDeliveryBabyFever = async (value) => {
-  return (await babyFeverField(value)).click();
+  const fever = await babyFeverField(value);
+  await fever.waitForClickable();
+  await fever.click();
 };
 
 const selectDeliveryBabyBlueSkin = async (value) => {
-  return (await babyBlueSkinField(value)).click();
+  const blueSkin = await babyBlueSkinField(value);
+  await blueSkin.waitForClickable();
+  await blueSkin.click();
 };
 
 const selectDeliveryPncVisits = async (value) => {
-  return (await deliveryPncVisitsField(value)).click();
+  const pncVisits = await deliveryPncVisitsField(value);
+  await pncVisits.waitForClickable();
+  await pncVisits.click();
 };
 
-const submitForm = async () => {
-  await (await submitButton()).click();
+const getSummaryInfo = async () => {
+  return {
+    sumPatientName: await sumPatientName().getText(),
+    sumPatientAge: await sumPatientAge().getText(),
+    sumWomanCondition: await sumWomanCondition().getText(),
+    sumDeliveryDate: await sumDeliveryDate().getText(),
+    sumDeliveryPlace: await sumDeliveryPlace().getText(),
+    sumDeliveredBabies: await sumDeliveredBabies().getText(),
+    sumDeceasedBabies: await sumDeceasedBabies().getText(),
+    sumPncVisits: await sumPncVisits().getText(),
```

Comment:
I thought about that, but in the end I decided to add the `sum` part to the name of the fields because if you only want the `patientName` for example, it might be confusing, what do you think? 

this is how it looks:

```.js
const { patientName }  = await deliveryForm.getSummaryInfo();
```

Do you think that it will be clear still in a scenario like this one ⬆️ ?

Reply:
If you need to customize the name when using it inline, you can always just override it!  

```
const { patientName: sumPatientName }  = await deliveryForm.getSummaryInfo();
```

Like I said before, this is not a big deal at all, but usually I do like to have the variable names make sense _in the context of the object that contains them_ and then consuming code can override the names as necessary....

Reply:
oh ok, it makes sense. 
I know that it is a minor change, but I still appreciate the comment, I will change it because I agree with what you are saying.
Thank you!